FastStyleTransfer/Models/TransformerNet.swift

```diff
@@ -0,0 +1,118 @@
+import TensorFlow
+import Foundation
+
+/// Model that applies style
```

Comment:
```suggestion
/// A model that applies style.
```
End every doc comment with a period.

Reply:
Done